elegant/utils.py

```diff
@@ -49,3 +49,10 @@ def wrapper(*args, **kwargs):
         except Exception:
             logging.error(traceback.format_exc())
     return wrapper
+
+
+def safe_repr(val, unit=1.0, fmt="{:.3g}"):
+    if val == np.inf:
+        return "\u221E"
+    else:
+        return fmt.format(val / unit)
```

Comment:
I suggest the following change, which makes code a little bit clean:

```
def safe_repr(args):
    if val == np.inf:
        return "\u221E"
    return fmt.format(val / unit)
```

Reply:
done